src/spec/alarm_table_model_spec.js

```diff
@@ -0,0 +1,63 @@
+import {TableModel} from '../panels/alarm-table/table_model';
+
+describe('TableModel', function() {
+
+  it('should work', function () {
```

Comment:
LOL

Reply:
Forget I left that there... I guess I'll keep it for lolz :)